### PR TITLE
utmp implementations from LFS

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: options/linux/generic/utmp-stubs.cpp [hunk 1]
```diff
@@ -1,27 +1,116 @@
...
+#include <errno.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <bits/ensure.h>
+#include <mlibc/debug.hpp>
+
+/*
+ * The code in this file is largely based on glibc.
```
Comment (Geertiebear) [inline options/linux/generic/utmp-stubs.cpp:11]: This is a license conflict.

**Review by Geertiebear — CHANGES_REQUESTED** [review]
Unfortunately we can't include GPL'd code in mlibc.

File: options/linux/generic/utmp-stubs.cpp (hunk 1 above)
Comment (no92) [reply options/linux/generic/utmp-stubs.cpp]: The offending commits were dropped.

**Comment by no92** [comment]
Dropped the offending commits and rebased onto master.

**Review by Dennisbonke — APPROVED** [review]
I believe that this is good to go now, not as extensive as it once was, but progress none the less and no more license violations! LGTM.

**Pull request merged**